week-1/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,16 @@
+function greeting(){
+    return "HELLO" + upperCase();
+}
+
 var mentor1 = "Daniel";
 var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+console.log(mentor1);
```

Comment:
Does this output the expected result? What about your function `greeting`? Try using it.